src/core/config.ts

```diff
@@ -1,21 +1,43 @@
-import { Context, PUSH_BASE_URL, PlatformConfig } from '.';
+import {
+    Context,
+    PUSH_BASE_URL,
+    PlatformConfig,
+    DDLPromptConfig,
+    PromptTypeName,
+    ReminderTimeUnit,
+    UiActionType,
+    PromptPosition,
+    DDL_BASE_URL
+} from '.';
 import { get, set } from './storage';
 
 import { authedFetchJson } from './utils';
 
-export async function loadConfig(ctx: Context): Promise<PlatformConfig> {
-    let config = (await get<PlatformConfig>('platformConfig')) ?? {};
+interface DDLConfigApiShape {
```

Comment:
Minor thing about naming, on most of the codebases we tend to make acronyms follow only first is upper (e.g. `DdlConfigApiShape`).

It's a convention we've carried from the server side naming conventions, but for consistency worth renaming instances in the SDK here too?

Reply:
Yeah, not sure why - I actually personally dislike caps in type names also - just residual from the spiking I think, never noted this one here but will rename it 👍